third_party/move/tools/move-spec-test/src/lib.rs

```diff
@@ -120,6 +139,9 @@ pub fn run_spec_test(
         }
     }
 
+    benchmarks.prover.stop();
+    benchmarks.prover_results.extend(proving_benchmark);
```

Comment:
I suggest renaming it to `proving_benchmarks`
```suggestion
    benchmarks.prover_results = proving_benchmarks;
```

Reply:
Done.